archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -34,36 +34,35 @@
             <groupId>com.paypal.selion</groupId>
             <artifactId>SeLion</artifactId>
             <version>${selion.version}</version>
+            <scope>test</scope>
```

Comment:
Perhaps I should leave this as it was for this artifact. `TestServerUtils` and `AppLogger` are the `application` under test for the purposes of the included sample code. The problem is they depend on downstream dependencies of the SeLion artifact -- requiring it (the SeLion artifact) to be in `compile` scope. Ideally, I'd like to convey that SeLion should be in `test` scope when used solely for writing/running tests. It should be in the `compile` scope only when creating a re-usable artifact that adds to / sits on-top of a SeLion one.

I suppose another option is to explicitly define the transitive `<dependencies>` of `TestServerUtils` and `AppLogger` in the `compile` scope.

@sebady your thoughts here?